PDP11/pdp11_rq.c

```diff
@@ -753,31 +753,35 @@ x  RA73 70(+1)  21      2667+   21      1       ?       3920490
 
 // AFAIK the UNIBUS KLESI and QBUS KLESI used the same controller type ...
 #define KLESI_CTYPE     1               // RC25 controller (UNIBUS and QBUS both)
-#define KLESI_UQPM      1
-#define KLESI_MODEL     1
+#define KLESI_UQPM      1               // Mass_Storage_Control_Protocol_Ver_2.4.0_Jun92.txt Appendix C:
+#define KLESI_MODEL     1               // 1 | HSC50; 3 | RC25 integrated controller

```

Comment:
That is (and was) incorrect: model 1 is HSC50, for RC25 you need 3.  If I change it to 3 RSTS reports it as RC25 (rather than "unknown" which is how it reports things that aren't plausible PDP-11 peripherals).